ClientTests/AdjustTelemetryHelperTests.swift

```diff
@@ -36,6 +35,7 @@ class AdjustTelemetryHelperTests: XCTestCase {
         XCTAssertFalse(GleanMetrics.Adjust.adGroup.testHasValue())
         XCTAssertFalse(GleanMetrics.Adjust.creative.testHasValue())
         XCTAssertFalse(GleanMetrics.Adjust.network.testHasValue())
+        XCTAssertFalse(GleanMetrics.Adjust.deeplinkReceived.testHasValue())
```

Comment:
We just set the attribution data we don't send the event so we shouldn't test the event value here.